config/toml.go

```diff
@@ -247,8 +247,9 @@ laddr = "{{ .P2P.ListenAddress }}"
 # Address to advertise to peers for them to dial
 # If empty, will use the same port as the laddr,
 # and will introspect on the listener or use UPnP
-# to figure out the address.
-external_address = "{{ .P2P.ExternalAddress }}"
+# to figure out the address. ip and port are required
+# example: 159.89.10.97:26656
+external-address = "{{ .P2P.ExternalAddress }}"
```

Comment:
```suggestion
external_address = "{{ .P2P.ExternalAddress }}"
```

Reply:
Was scratching my head as to why this wasn't working but I think we need to keep the underscore else this doesn't work